providers/resource/bin/graphics_card_resource.py

```diff
@@ -23,6 +23,25 @@
 import subprocess
 import shlex
 import string
+import re
+
+
+def get_ubuntu_version():
+    """Get Ubuntu release version for checking."""
+    try:
+        import distro
+        return float(distro.version())
```

Comment:
Why returning a float instead of a string?

```
>>> float('22.10') > float('22.04')
True
>>> '22.10' > '22.04'
True
```

Reply:
I used your method at first, however it return wrong value. I'll check this again.

Reply:
Thanks for your notice. I find [packaging.version.parse](https://packaging.pypa.io/en/stable/version.html) could do this. However we still have to notice some bugs described in this  [stack overflow](https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python). I'll test it to make sure everything goes well.